app/javascript/react/screens/App/Plan/PlanReducer.js

```diff
@@ -70,7 +67,9 @@ const _formatPlanRequestDetails = data => {
       if (taskDetails.completed) {
         taskDetails.completedSuccessfully =
           task.options.progress.current_description ===
-          'Virtual machine migrated';
+            'Virtual machine migrated' ||
+          task.options.progress.current_description ===
+            'Mark source as migrated';
```

Comment:
@priley86 I was playing with the one of the opentlc lab setups... and noticed that `task.options.progress.current_description` was `'Mark source as migrated'` for successfully completed tasks.
(which was completely unexpected)
I have added that as a check as well for `taskDetails.completedSuccessfully` to be `true`